waspc/data/Generator/templates/react-app/index.html

```diff
@@ -3,7 +3,11 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="shortcut icon" href="/favicon.ico" />
+
+        {=#isFaviconPresent=}
```

Comment:
```suggestion
        {=# isFaviconPresent =}
```

Throughout the templates, we used spaces around the variable names.